### PR TITLE
Add an example room air conditioner configuration to chef devices

### PR DESCRIPTION
**Pull request by mideayanghui**
-   Adds the room air conditioner device type to the list of matter device type.
- Adds an example room air conditioner device type configuration in chef/devices.

### CODE REVIEW

File: examples/chef/devices/rootnode_roomairconditioner_9cf3607804.matter [hunk 1]
```diff
@@ -0,0 +1,1400 @@
...
+  readonly attribute event_id eventList[] = 65530;
+  readonly attribute attrib_id attributeList[] = 65531;
+  readonly attribute bitmap32 featureMap = 65532;
+  readonly attribute int16u clusterRevision = 65533;
+}
+
+endpoint 0 {
+  device type rootdevice = 22, version 1;
+
+  server cluster Groups {
```
Comment (bzbarsky-apple) [inline examples/chef/devices/rootnode_roomairconditioner_9cf3607804.matter:1179]: Why does this have groups on EP0?

File: examples/chef/devices/rootnode_roomairconditioner_9cf3607804.matter [hunk 2]
```diff
@@ -0,0 +1,1400 @@
...
+    callback attribute bootReason;
+    callback attribute activeHardwareFaults;
+    callback attribute activeRadioFaults;
+    callback attribute activeNetworkFaults;
+    callback attribute testEventTriggersEnabled default = false;
+    ram      attribute featureMap default = 0;
+    ram      attribute clusterRevision default = 0x0001;
+  }
+
+  server cluster Switch {
```
Comment (bzbarsky-apple) [inline examples/chef/devices/rootnode_roomairconditioner_9cf3607804.matter:1277]: Why does this have a Switch server on EP0?

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]

File: examples/chef/devices/rootnode_roomairconditioner_9cf3607804.matter (hunk 2 above)
Comment (mideayanghui) [reply examples/chef/devices/rootnode_roomairconditioner_9cf3607804.matter:1277]: Remove

File: examples/chef/devices/rootnode_roomairconditioner_9cf3607804.matter (hunk 1 above)
Comment (mideayanghui) [reply examples/chef/devices/rootnode_roomairconditioner_9cf3607804.matter:1179]: Remove. Just copy ./examples/chef/devices/template.zap and modify

**Review by bzbarsky-apple — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Pull request merged**